wordpress-mobile/WordPress-Android
# Optimize Jetpack badge margin on me screen

This optimizes Jetpack badge placement on me screen according to different screen sizes. On most popular Android devices (e.g., Galaxy S21, Galaxy S22), the Jetpack badge is slightly cut off, you need to scroll to see the whole badge. This PR reduces the top margin of the Jetpack badge for phones but keeps it larger for tablets.
[Internal discussion](https://apptestingo2.wordpress.com/2022/07/26/to-test-wordpress-android-20-4/#comment-13181)

This is a quick and cheap solution for handling cut-off badge. If a button is added/removed on me screen, `/values-h720dp` should be updated too.

|before|after|
|-|-|
|<img src="https://user-images.githubusercontent.com/2471769/186958055-3d949ac8-fe7c-4821-aa63-33563de70b60.jpg" width=240>|<img src="https://user-images.githubusercontent.com/2471769/186958114-4586a40d-226f-409c-942d-6402ff4de53d.jpg" width=240>|
|<img src="https://user-images.githubusercontent.com/2471769/186958244-ba563967-1547-40cd-80c2-65e889118377.png" width=320>|<img src="https://user-images.githubusercontent.com/2471769/186958277-a9c36556-4281-4782-ad62-73ac1c7ecf81.png" width=320>|


To test:
1. Launch the app.
2. Log in if necessary.
3. Open me screen by tapping your profile icon top right of the screen.
4. Notice Jetpack badge is not cut off on short screens.
5. Repeat 1-4 on different screen sizes and ensure the badge looks okay on the layout.

## Regression Notes
1. Potential unintended areas of impact
The Jetpack badge margins on different screens.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Tested on different size emulators.

3. What automated tests I added (or what prevented me from doing so)
It's a small UI change. I didn't add any tests.

PR submission checklist:
